client/signup/steps/theme-selection/index.jsx

```diff
@@ -59,7 +48,8 @@ module.exports = React.createClass( {
 	},
 
 	getThemes() {
-		return this.props.signupDependencies.themes || this.props.themes;
+		const dependencies = this.props.signupDependencies;
+		return dependencies.themes || getThemes( dependencies.surveyQuestion, dependencies.designType );
```

Comment:
What happens here if `dependencies.themes`, `dependencies.surveyQuestion`, and `dependencies.designType` are all null?